src/CMakeLists.txt

```diff
@@ -9,6 +9,9 @@ include(BuildLGraphServer.cmake)
 include(BuildCypherLib.cmake)
 include(BuildClients.cmake)
 include(BuildCythonExtension.cmake)
+if (ENABLE_C_BINDINGS)
```

Comment:
remove the if branch, enable always

Reply:
At most of the time, users have no need to compile c bindings. The if-branch might reduce compile time.

Reply:
The `include/lgraph/c.h` will always be provided, but the functions defination might miss when turn off the option here. So always turning this on is a better idea.

Reply:
OK, follow the tugraph convention.

Reply:
I have all if-branch in changed CMakeLists.txt and remove ENABLE_C_BINDING option in Option.cmake